### PR TITLE
[FLINK-16128][docs]Translate "Google Cloud PubSub" page into Chinese

### PR DESCRIPTION
**Pull request by Sxnan**
## What is the purpose of the change

This pull request translates [Google Cloud PubSub](https://ci.apache.org/projects/flink/flink-docs-master/zh/dev/connectors/pubsub.html) page of connectors into Chinese

## Brief change log

- Translate the markdown file located at flink/docs/dev/connectors/pubsub.zh.md

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): no
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: no
  - The serializers: no
  - The runtime per-record code paths (performance sensitive): no
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Yarn/Mesos, ZooKeeper: no
  - The S3 file system connector: no

## Documentation

  - Does this pull request introduce a new feature? no
  - If yes, how is the feature documented? not applicable

### CODE REVIEW

File: docs/dev/connectors/pubsub.zh.md [hunk 1]
```diff
@@ -143,22 +135,22 @@ env.addSource(pubsubSource)
...
+另一个可能的原因是超过了确认的截止时间。这是收到信息的间隔和信息确认的间隔。PubSubSource 只有在信息被成功快照之后才会确认以保证至少一次的语义。这意味着，如果你的快照间隔大于信息确认的截止时间，那么你订阅的信息很有可能会被多次处理。
 
-For this reason it's recommended to have a (much) lower checkpoint interval than acknowledgement deadline.
+因此，我们建议把快照的间隔设置得比信息确认截止时间更短。
 
-See [PubSub](https://cloud.google.com/pubsub/docs/subscriber) for details on how to increase the acknowledgment deadline of your subscription.
+参照 [PubSub](https://cloud.google.com/pubsub/docs/subscriber) 来增加信息确认截止时间。
 
-Note: The metric `PubSubMessagesProcessedNotAcked` shows how many messages are waiting for the next checkpoint before they will be acknowledged.
+注意: `PubSubMessagesProcessedNotAcked` 显示了有多少信息正在等待下一个快照还没被确认。
```
Comment (PatrickRen) [inline docs/dev/connectors/pubsub.zh.md]: I think we don't need to translate "checkpoint" according to Flink Translation Specifications

File: docs/dev/connectors/pubsub.zh.md [hunk 2]
```diff
@@ -143,22 +135,22 @@ env.addSource(pubsubSource)
...
 
 #### SinkFunction
 
-The sink function buffers messages that are to be send to PubSub for a short amount of time for performance reasons. Before each checkpoint this buffer is flushed and the checkpoint will not succeed unless the messages have been delivered to PubSub.
+Sink function 会把准备发到 PubSub 的信息短暂地缓存以提高性能。每次快照前，它会 flush 缓冲区，并且只有当所有信息成功发送到 PubSub 之后，快照才会成功完成。
```
Comment (PatrickRen) [inline docs/dev/connectors/pubsub.zh.md]: Same issue of "checkpoint" here. I think "flush" can be translated as "刷新".

File: docs/dev/connectors/pubsub.md [hunk 3]
```diff
@@ -143,13 +143,13 @@ env.addSource(pubsubSource)
...
 
-### Atleast once guarantee
+### At least once guarantee
 
 #### SourceFunction
 
 There are several reasons why a message might be send multiple times, such as failure scenarios on Google PubSub's side.
 
-Another reason is when the acknowledgement deadline has passed. This is the time between receiving the message and between acknowledging the message. The PubSubSource will only acknowledge a message on successful checkpoints to guarantee Atleast-Once. This does mean if the time between successful checkpoints is larger than the acknowledgment deadline of your subscription messages will most likely be processed multiple times.
+Another reason is when the acknowledgement deadline has passed. This is the time between receiving the message and between acknowledging the message. The PubSubSource will only acknowledge a message on successful checkpoints to guarantee at-least-Once. This does mean if the time between successful checkpoints is larger than the acknowledgment deadline of your subscription messages will most likely be processed multiple times.
```
Comment (PatrickRen) [inline docs/dev/connectors/pubsub.md]: I think using lower case for all three words is better: "at-least-once"

File: docs/dev/connectors/pubsub.zh.md [hunk 4]
```diff
@@ -23,9 +23,7 @@ specific language governing permissions and limitations
 under the License.
 -->
 
-This connector provides a Source and Sink that can read from and write to
-[Google Cloud PubSub](https://cloud.google.com/pubsub). To use this connector, add the
-following dependency to your project:
+这个连接器可向 [Google Cloud PubSub](https://cloud.google.com/pubsub) 读取与写入数据。添加下面的依赖来使用此连接器:
```
[the remark this answers is not in the document]
Comment (Sxnan) [reply docs/dev/connectors/pubsub.zh.md]: I am not translating the Source and Sink"  to keep it consistent with the opening of other connectors such as https://ci.apache.org/projects/flink/flink-docs-master/zh/dev/connectors/filesystem_sink.html

File: docs/dev/connectors/pubsub.zh.md [hunk 5]
```diff
@@ -39,24 +37,17 @@ following dependency to your project:
 <b>Note</b>: This connector has been added to Flink recently. It has not received widespread testing yet.
```
[the remark this answers is not in the document]
Comment (Sxnan) [reply docs/dev/connectors/pubsub.zh.md]: Good catch!

File: docs/dev/connectors/pubsub.zh.md [hunk 6]
```diff
@@ -39,24 +37,17 @@ following dependency to your project:
...
-Google PubSub has an `at-least-once` guarantee and as such the connector delivers the same guarantees.
+连接器可以接收和发送 Google PubSub 的信息。和 Google PubSub 一样，这个连接器能够保证`至少一次`的语义。
 
 ### PubSub SourceFunction
 
-The class `PubSubSource` has a builder to create PubSubsources: `PubSubSource.newBuilder(...)`
+`PubSubSource` 类的对象由构建类来构建: `PubSubSource.newBuilder(...)`
 
-There are several optional methods to alter how the PubSubSource is created, the bare minimum is to provide a Google project, Pubsub subscription and a way to deserialize the PubSubMessages.
+除了 Google project，Pubsub subscription 和反序列化 PubSubMessage的方法是必须的，你有多种可选的方法来创建 PubSubSource。
```
[the remark this answers is not in the document]
Comment (Sxnan) [reply docs/dev/connectors/pubsub.zh.md]: Thanks for the suggestion

File: docs/dev/connectors/pubsub.md (hunk 3 above)
Comment (Sxnan) [reply docs/dev/connectors/pubsub.md]: nice catch!

File: docs/dev/connectors/pubsub.zh.md (hunk 2 above)
Comment (Sxnan) [reply docs/dev/connectors/pubsub.zh.md]: Thanks

File: docs/dev/connectors/pubsub.zh.md [hunk 7]
```diff
@@ -143,22 +135,22 @@ env.addSource(pubsubSource)
...
-### Atleast once guarantee
+### 精确一次语义保证
 
 #### SourceFunction
 
-There are several reasons why a message might be send multiple times, such as failure scenarios on Google PubSub's side.
+有很多原因导致会一个信息会被多次发出，例如 Google PubSub 的故障。
 
-Another reason is when the acknowledgement deadline has passed. This is the time between receiving the message and between acknowledging the message. The PubSubSource will only acknowledge a message on successful checkpoints to guarantee Atleast-Once. This does mean if the time between successful checkpoints is larger than the acknowledgment deadline of your subscription messages will most likely be processed multiple times.
+另一个可能的原因是超过了确认的截止时间。这是收到信息的间隔和信息确认的间隔。PubSubSource 只有在信息被成功快照之后才会确认以保证至少一次的语义。这意味着，如果你的快照间隔大于信息确认的截止时间，那么你订阅的信息很有可能会被多次处理。
```
[the remark this answers is not in the document]
Comment (Sxnan) [reply docs/dev/connectors/pubsub.zh.md]: Thanks, I will update the english version as well

**Comment by rmetzger** [comment]
@PatrickRen Let me know when the PR is in a good shape. I'm happy to merge the PR once it's ready.

File: docs/dev/connectors/pubsub.zh.md [hunk 8]
```diff
@@ -36,27 +34,20 @@ following dependency to your project:
...
 </p>
 
-Note that the streaming connectors are currently not part of the binary
-distribution. See
-[here]({{ site.baseurl }}/dev/projectsetup/dependencies.html)
-for information about how to package the program with the libraries for
-cluster execution.
-
-
+注意连接器目前还不是二进制发行版的一部分，添加依赖、打包配置以及集群运行信息请参考[这里]({{ site.baseurl }}/dev/projectsetup/dependencies.html)
```
Comment (wuchong) [inline docs/dev/connectors/pubsub.zh.md]:
```suggestion
注意连接器目前还不是二进制发行版的一部分，添加依赖、打包配置以及集群运行信息请参考[这里]({{ site.baseurl }}/zh/dev/projectsetup/dependencies.html)
```

File: docs/dev/connectors/pubsub.zh.md [hunk 9]
```diff
@@ -143,22 +135,22 @@ env.addSource(pubsubSource)
 </div>
 </div>
 
-### Atleast once guarantee
+### 精确一次语义保证
```
Comment (wuchong) [inline docs/dev/connectors/pubsub.zh.md]:
```suggestion
### 至少一次语义保证
```

**Pull request merged**